starknet-replay/src/runner/replace_ids.rs

```diff
@@ -150,6 +154,55 @@ mod tests {
         fs::read_to_string(sierra_program_json_file)
     }
 
+    // This is because the built-in equality doesn't check for matching `debug_name`
+    // string.
+    fn compare_libfunc_declaration(a: &[LibfuncDeclaration], b: &[LibfuncDeclaration]) -> bool {
```

Comment:
Compare to me implies returning an ordering. I think this is an equality check. I suggest renaming to `libfunc_declaration_eq`.